### DISCUSSION TITLE
Is this the expected block style behaviour?

### DISCUSSION THREAD

**Discussion by mccarthy-james** [mikefarah/yq#696] [Q&A] [answered]
I have some yaml with the following block style:

```
a:>-
  some text
  goes here
```
  
When yq processes it I get:

```
a:>-
  some text  goes here
```

Of course, this how it is supposed to be interpreted, but as this is an instruction for the destination application and not yq, I do not yq to modify the yaml in this way. Is this a reasonable expectation or am I using yq in a way it is not designed to be used? Perhaps that could be toggled?

With `a:|-` there is no change, as expected, but my application requires `a:>-`.

**Comment by mikefarah** [comment]
This is the result of the underlying yaml parser, which occasionally mangles white space), however generally this only affects human readability of the yaml file, not processing. TLDR: The two string values should be equivalent (ref https://yaml-multiline.info/)

**Comment by mccarthy-james** [comment, marked as answer]
Thanks, Mike. This is editing a docker-compose.yml file and it seems that it doesn't matter for processing after all (I thought I had tested and it had). I also switched the block style to to `|-` and everything worked as expected still! So that might be the better solution. Thanks again.
